### DISCUSSION TITLE
Include empty lines in flashcards

### DISCUSSION THREAD

**Discussion by Maano26** [st3v3nmw/obsidian-spaced-repetition#395] [Q&A]
Hey everyone,

is there any way to include empty lines into the flashcards?

For example

> 
> This is the question #flashcards
> ?
> This line is included
> 
> This line is currently not included
> But I would love include it as well

Usually the answers to my flashcards a quite long and thus including empty lines massivley improves their readibility. If empty lines cannot be included, is there some workaround to improve readability of long cards?

Thank you!

**Comment by dev-msp** [comment]
Use blockquotes:

```
This is the question #flashcards
?
> This line is included
> 
> This line is currently not included
> But I would love include it as well
```

And if it results in styling issues, it should be straightforward to write a little CSS that adjusts how blockquotes look in a flashcard popup.
